### PR TITLE
Add TripleWhale / Ecommerce

### PR DESCRIPTION
**Pull request by Overload119**
### website
https://triplewhale.com/

### examples
```
https://thehermoza.com/
https://freshk9.com/
https://www.glamrdip.com/
https://hoplark.com/
https://myobvi.com/
```

### CODE REVIEW

File: src/technologies/t.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 {
+  "TripleWhale": {
+    "cats": [
+      10
+    ],
+    "html": "triplewhale-pixel.web.app\/triplepx.txt",
```
Comment (nurbek91) [inline src/technologies/t.json:6]: The html parameter is resource intensive and can be used when all other methods are not possible.
In your case, we can replace it with 
 "xhr": "triplewhale-pixel\\.web\\.app",

File: src/technologies/t.json [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+    "saas": true,
+    "icon": "triplewhale.png",
```
Comment (nurbek91) [inline src/technologies/t.json:8]: Preferred icon format is svg with size 32x32
[Triple Whale.svg.zip](https://github.com/wappalyzer/wappalyzer/files/9264825/Triple.Whale.svg.zip)

File: src/technologies/t.json [hunk 3]
```diff
@@ -1,4 +1,13 @@
 {
+  "TripleWhale": {
```
Comment (nurbek91) [inline src/technologies/t.json:2]: The correct name of technologies is two words written separately? Triple Whale

We can add description:

`"description": "Triple Whale is a mobile app used by ecommerce businesses to track profits, blended ROAS, LTV, and much more, integrating into a single easy-to-use mobile app.",`

pricing:
```
"pricing": [
      "mid",
      "recurring"
    ],
```

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!
Needs an update to perfect your PR

File: src/technologies/t.json (hunk 3 above)
Comment (Overload119) [reply src/technologies/t.json:2]: That description is not quite right - Triple Whale has an app, but it's not known as that.
I will pull a description from Crunchbase.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**